source/configuring_domain/01_managing_security/saml.rst

```diff
@@ -1,70 +1,87 @@
 Single sign-on with SAML
 ========================
 
-.. admonition:: Caution
-   :class: caution
+.. admonition:: Note
+   :class: note
 
    The availability of this feature depends on the license of the Opendatasoft domain.
 
 
-Register your SAML identity provider on your domain
----------------------------------------------------
+Registering your SAML identity provider on your domain
+------------------------------------------------------
 
-1. Navigate to the signup page in the domain configuration interface.
+Step 1: Allow access for SAML users
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-2. Check "Allow access for SAML users".
+Enable SAML so that authentication for your portal users can be delegated to your own identity service.
+
+1. From the back office, go to **Configuration > Signup**.
+
+2. Toggle on **Allow access for SAML users**.
+
+A list of configuration settings appears:
 
 .. image:: images/configuration_SAML.png
     :alt: SAML identity provider configuration interface
 
-3. Paste your identity provider metadata document in the "Identity provider metadata document" field.
+Step 2: Configure SAML settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-4. If your identity provider is Microsoft Azure Active Directory, check the related checkbox.
+1. Paste your identity provider metadata document into the **Identity provider (idp) metadata document** field.
 
-5. If you would like to disable local user creation, making sure only existing users can connect to the platform through SAML, check the "Disable local user provisioning" checkbox.
+2. If you want to use the ``FriendlyName`` attribute for unidentified attributes, toggle on **Use friendly name for attribute mapping**.
```

Comment:
I'm not sure we should document this specific option, this option is visible only on domain which had SAML already enabled (so not for new domains and not for domains that will get SAML from now on). So it think this could be handled by support only.

Reply:
It makes sense. I'm removing this instruction.

Reply:
✅